from hate.logger import logging
import sys
from hate.exception import CustomException

# try:
#     # Example code that causes an error
#     1 / "0"
# except Exception as e:
#     # Raise a custom exception
#     raise CustomException("An error occurred", sys)


#logging.info("Welcome")